spec/rails_helper.rb

```diff
@@ -41,6 +41,10 @@
 #
 Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
 
+# loads Knapsack factories
+knapsack_factory_path = Rails.root.join('..', 'spec', 'factories', '*.rb')
```

Comment:
If you use the knapsack path (see lib/hyku_knapsack.rb) you don’t need to deal with this relative path thing. Also you should handle us directories with ** and lastly you don’t need the if statement if you are doing an each. Dir[i_dont_exist] == [] and [].each is a no op already

Reply:
Also, why do you need factories from the knapsack in the main app? Aren’t knapsack tests run with their own rails helper in the directory one up?

Reply:
If that's then case then i must not be running the specs correctly, i just bash into my web container and run `rspec` so if there's something else I need to do then my mistake.

https://github.com/samvera/hyku/blob/load-knapsack-factories/spec/abilities/work_ability_spec.rb#L17
in particular was running through all the work types and erroring out because there were no factories for Adventist work types in Hyku